dusty/systems/docker/testing_image.py

```diff
@@ -0,0 +1,43 @@
+from . import get_docker_client
+
+
+def _ensure_testing_spec_base_image(testing_spec):
+    if 'image' in testing_spec and 'build' in testing_spec:
+        raise RuntimeError('Only 1 of `image` and `build` keys are allowed in testing spec')
+    elif 'image' in testing_spec:
+        return testing_spec['image']
+    elif 'build' in testing_spec:
+        docker_client = get_docker_client()
+        image_tag = 'dusty_testing_base/image'
```

Comment:
this only sticks around for a short period of time, until the new image is build off of this base image.  If this already exists it is overwritten.